lib/elixir/test/elixir/test_helper.exs

```diff
@@ -1,4 +1,10 @@
-ExUnit.start [trace: "--trace" in System.argv]
+exclude =
+  case :erlang.system_info(:otp_release) do
```

Comment:
For future cases like this I think we should stick to using application versions because people can build custom versions of OTP that don't match an OTP version.